core/src/qos_service.rs

```diff
@@ -153,10 +153,15 @@ impl QosService {
                             self.metrics.retried_txs_per_account_limit_count.fetch_add(1, Ordering::Relaxed);
                             Err(TransactionError::WouldExceedMaxAccountCostLimit)
                         }
-                        CostTrackerError::WouldExceedAccountDataMaxLimit => {
+                        CostTrackerError::WouldExceedAccountDataBlockLimit => {
                             self.metrics.retried_txs_per_account_data_limit_count.fetch_add(1, Ordering::Relaxed);
                             Err(TransactionError::WouldExceedMaxAccountDataCostLimit)
                         }
+                        CostTrackerError::WouldExceedAccountDataTotalLimit => {
+                            // NOTE: Do *not* increment a retry counter, since these
+                            // transactions will *not* be retried
+                            Err(TransactionError::WouldExceedMaxTotalAccountDataCostLimit)
```

Comment:
Can we keep both these error names the same, one or the other

Reply:
Done!